packages/manager/modules/log-to-customer/src/list/component.js

```diff
@@ -13,5 +13,7 @@ export default {
     goBack: '<',
     title: '@',
     description: '@',
+    trackingHits: '<',
+    trackClick: '<',
```

Comment:
Where is defined the root `trackClick` function ? I don't see it in your pull request ?

Reply:
the function is passed through the parent (see audit-logs.html)

Reply:
Which file exactly ?